examples/custom-reducer/package.json

```diff
@@ -8,8 +8,7 @@
     "kepler.gl": "^1.0.0-2",
     "react": "^16.8.4",
     "react-dom": "^16.8.4",
-    "react-palm": "^1.1.2",
```

Comment:
we don't need this dependency since we are using enhanceReduxMiddleware which will inject react-palm into application consumer